lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -277,4 +277,29 @@ export class ServerRequestParameters {
     static isSSOParam(request: AuthenticationParameters) {
         return request && (request.account || request.sid || request.loginHint);
     }
+
+    /**
+     * Returns the correct response_type string attribute for an acquireToken request configuration
+     * @param accountsMatch boolean: Determines whether the account in the request matches the cached account
+     * @param scopes Array<string>: AuthenticationRequest scopes configuration
+     * @param loginScopesOnly boolean: True if the scopes array ONLY contains the clientId or any combination of OIDC scopes, without resource scopes
+     */
+    static determineResponseType(accountsMatch: boolean, scopes: Array<string>, loginScopesOnly: boolean) {
```

Comment:
`setResponseType()`?

Reply:
I think calling it setResponseType would be a misnomer, since "set" is usually an attribute mutator and this method returns a value without mutating the actual request params.